.github/workflows/push.yml

```diff
@@ -81,8 +81,8 @@ jobs:
       -
         name: Run tests
         run: |
-          docker-compose run wait
-          docker-compose run tests
+          docker compose run wait
```

Comment:
tohle se snažím likvidovat, kde na to narazím, ale není to notný řešit tu (např. https://github.com/keboola/db-extractor-mysql/pull/179/commits/ee98caea692ea356d95c6f2ee83b506153d61538)

Reply:
ok, rovnou to upravím

Reply:
https://github.com/keboola/db-writer-mssql/pull/78/commits/10d80d5e9381f8551fc12c01dc248ea075b52022